Source/Renderer/Context.js

```diff
@@ -271,7 +271,10 @@ define([
         this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']);
         this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
         this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
-        this._debugShaders = getExtension(gl, ['WEBGL_debug_shaders']);
+        this._debugShaders = !!getExtension(gl, ['WEBGL_debug_shaders']);
+        this._s3tc = !!getExtension(gl, ['WEBGL_compressed_s3tc', 'MOZ_WEBGL_compressed_texture_s3tc', 'WEBKIT_WEBGL_compressed_texture_s3tc']);
```

Comment:
Here and below, do you know if the `'MOZ_`and `WEBKIT_` prefixes are in widespread use?  If this is just for older browsers, we can drop them.